embulk-input-mysql/src/test/java/org/embulk/input/mysql/IncrementalTest.java

```diff
@@ -104,4 +104,24 @@ public void testTimestamp() throws Exception
         assertThat(readSortedFile(out2), is(readResource("ts/expected_2.csv")));
         assertThat(result2.getConfigDiff(), is((ConfigDiff) loadYamlResource(embulk, "ts/expected_2.diff")));
     }
+
+    @Test
+    public void testQueryWithPlaceholder() throws Exception
```

Comment:
Can you add a test for multiple incremental columns?

Reply:
Fixed at 9fe5f87b02e26b2f3b27b0cae7238b8d38e62c58 and added for embulk-input-(oracle|db2|sqlserver|postgresql|oracle|redshift) dc0f798c668eb44e2ced2075ddf5c882e3b48d3a